pkgs/applications/science/physics/yade/default.nix

```diff
@@ -0,0 +1,121 @@
+with (import <nixpkgs> {});
+
+let 
+
+  pythonPackages = python27Packages;
+
+    minieigen = pythonPackages.buildPythonPackage rec {
+      name = "minieigen";
+
+      src = pkgs.fetchurl {
+        url = "https://github.com/eudoxos/minieigen/archive/master.zip";
```

Comment:
This is not a stable url. Either refer to a commit or a release. Also use `fetchFromGitHub`.